frontend/packages/ceph-storage-plugin/integration-tests/views/obc.view.ts

```diff
@@ -0,0 +1,50 @@
+import { $, $$, browser, ExpectedConditions as until } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import * as sideNavView from '@console/internal-integration-tests/views/sidenav.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { CLAIM_STATUS } from '../utils/consts';
+import { ObcType, selectItemFromDropdown } from '../utils/helpers';
+
+// create obc
+export const namespaceDropdown = $$('[class="pf-c-dropdown__toggle pf-m-plain"]').get(0);
```

Comment:
Will this always be the `0`th element?  I will prefer to have the alternative approach as this approach is not very reliable.

Reply:
Thanks, I made the changes in obc and pvc views to get rid of it.